src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -302,6 +303,20 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
         error.addToHolder(holder)
     }
 
+    private fun checkUnstableAttribute(ref: RsReferenceElement, o: RsElement, holder: RsAnnotationHolder) {
+        val element = ref.reference.resolve() as? RsOuterAttributeOwner ?: return
+        if (element.containingCargoPackage?.origin == PackageOrigin.STDLIB) return
```

Comment:
This line makes whole inspection useless because `unstable` attribute can be used only in stdlib. So `element.containingCargoPackage?.origin == PackageOrigin.STDLIB` should be always `true `